pyperf/base_test_results/test1/verify

```diff
@@ -0,0 +1,91 @@
+Test:Avg:Unit
+2to3:[0-9.]{1,}:ms|us|ns|s
+async_generators:[0-9.]{1,}:ms|us|ns|s
```

Comment:
Verification is fragile, tests change from version to version.

Improve regex `(m|u|n)?s`